bim2sim/tasks/bps/plot_results.py

```diff
@@ -38,7 +40,7 @@ class PlotBEPSResults(ITask):
     reads = ('df_finals', 'sim_results_path', 'ifc_files', 'elements')
     final = True
 
-    def run(self, df_finals, sim_results_path, ifc_files, elements):
+    def run(self, df_finals: dict, sim_results_path: Path, ifc_files: list, elements: dict):
```

Comment:
I got the "NoneType" for the parameters "df_finals" and "sim_results_path" when I debug, but acoording the Args above, I just add the `dict` and `Path`. 
![image](https://github.com/BIM2SIM/bim2sim/assets/159027366/8fc2681d-4203-417f-8458-f6c6d0200e6e)

Reply:
this is fine